eng/publishing/UpdateAkaMSLinks/UpdateAkaMSLinks.csproj

```diff
@@ -0,0 +1,72 @@
+<!-- Copyright (c) Microsoft Corporation. Use of this source code is governed by a BSD-style license that can be found in the LICENSE file. -->
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <!--
+    This is a 'csproj' even though it has no C# code. It is simply a csproj to take advantage of the
+    .NET SDK's default csproj handling, which lets us easily download the Links package and use its
+    task. Our target hooks into "Build", so "dotnet build" should be used to run this project in a
+    way that automatically restores the Links package and runs our target.
+  -->
+
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Versions.props))" />
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.Deployment.Tasks.Links" Version="$(MicrosoftDotNetDeploymentTasksLinksVersion)" />
+  </ItemGroup>
+
+  <!-- Fix Links package issue: wrong task name. https://github.com/dotnet/arcade/issues/8573 -->
+  <UsingTask
+    TaskName="CreateAkaMSLinks"
+    AssemblyFile="$(MicrosoftDotNetDeploymentTasksLinksTaskAssembly)"
+    Condition="'$(MicrosoftDotNetDeploymentTasksLinksTaskAssembly)' != ''" />
```

Comment:
When would this not be defined?

Reply:
During the Restore target/phase. `dotnet build` runs Restore, then Build in separate MSBuild contexts, and the props file inside the package that defines this property is only imported during Build.